rules/lib/is-promise.js

```diff
@@ -2,37 +2,35 @@
  * Library: isPromise
  * Makes sure that an Expression node is part of a promise.
  */
-var STATIC_METHODS = [
```

Comment:
I extracted this array into its own module so that I could use it in multiple files.